### PR TITLE
ath10k-firmware: add ath10k-firmware-qca9888-ct

### PR DESCRIPTION
**Pull request by tinnnysu**
add firmware support for QCA9888 ct firmware, tested on WLE650V5-18A

### CODE REVIEW

File: package/firmware/ath10k-firmware/Makefile [hunk 1]
```diff
@@ -178,6 +194,13 @@ $(Package/ath10k-firmware-default)
   CATEGORY:=Firmware
 endef
 
+define Package/ath10k-firmware-qca9888-ct
+$(Package/ath10k-firmware-default)
+  TITLE:=ath10k CT 10.4.3 firmware for QCA9988 devices
```
Comment (heinzek) [inline package/firmware/ath10k-firmware/Makefile]: QCA9988? Typo?

Comment (tinnnysu) [inline package/firmware/ath10k-firmware/Makefile]: Sorry for that. Now I fix it, https://github.com/lede-project/source/pull/1088/commits/78d9bc7259673eb1e8f07a77fe25dd448259ac8f Thanks

**Comment by pepe2k** [comment]
Hello @tinnnysu,

> Signed-off-by: sushitao <tinnnysu@gmail.com>

Please, read our [submitting patches rules](https://lede-project.org/submitting-patches), and make sure your patch or pull request follows them.

Cheers,
Piotr

**Comment by tinnnysu** [comment]
Hi @pepe2k,
Is it the name syntax error? This is a Chinese name. Would the following one to be correct? Sorry for my poor English.

> Signed-off-by: Shitao Su <tinnnysu@gmail.com>

**Comment by pepe2k** [comment]
Hi @tinnnysu,

Yes, we need real name and e-mail address in Signed-off-by line, thanks for fixing this!

Cheers,
Piotr

**Comment by blogic** [comment]
i have merged the patch that ben sent on the mailing list in favour of this one

**Pull request closed without merging**